superset-frontend/cypress-base/cypress/e2e/dashboard/drillby.test.ts

```diff
@@ -56,15 +56,21 @@ const drillBy = (targetDrillByColumn: string, isLegacy = false) => {
 
   cy.get('.ant-dropdown:not(.ant-dropdown-hidden)')
     .first()
-    .find("[role='menu'] [role='menuitem'] [title='Drill by']")
+    .should('be.visible')
+    .find("[role='menu'] [role='menuitem']")
+    .contains(new RegExp(`^Drill by$`))
```

Comment:
Nit, use `//` for static regexes as it's more readable IMHO:

```suggestion
    .contains(/^Drill by$/)
```